### DISCUSSION TITLE
Error pages doesn't shown

### DISCUSSION THREAD

**Discussion by ademsenel** [Dotnet-Boxed/Templates#68] [Q&A]
Hİ Rehan

i am using boilerplate source code (not extension) include elmah source code and nwebsec source code 

when i tried to typed wrong url it doesn't show error pages or something went wrong it doesn't show internal error pages in my project 

i checked everything and i could say noting is wrong in code 
also i checked with extension boilerplate project and the extension project is shown error page but not in my project 

do u have any idea 

best regards
Adem

**Comment by RehanSaeed** [comment]
Is this MVC 5 or 6?

**Comment by ademsenel** [comment]
hi 
this is mvc 5 project i found some solution but it was different from your project 
and i realy go crayz 

my solution is i change the httpErrors existingResponse section

your orginal code is "<httpErrors errorMode="Custom" existingResponse="PassThrough">" and work perfect but my project doesnt work 

so i change existingResponse section is <httpErrors errorMode="Custom" existingResponse="Replace"> 
and now its working 

i just know about what is wrong with my project

thank you so much your perfect solution when i finished my project if you would like i should serve to you

**Comment by RehanSaeed** [comment]
Post a link, I'll add it to the home page.

**Comment by RehanSaeed** [comment]
Feel free to re-open if you come back.
